xml/System.Runtime.InteropServices/ICustomMarshaler.xml

```diff
@@ -125,7 +125,10 @@ const IID IID_IOld = {0x9B2BAADD,0x0705,0x11D3,{0xA0,0xCD,0x00,0xC0,0x4F,0xA3,0x
  <xref:System.Runtime.InteropServices.ICustomMarshaler.GetNativeDataSize%2A?displayProperty=nameWithType>  
   
  Returns the size of the unmanaged data to be marshaled.  
-  
+
+> [!NOTE]
+>  If a custom marshaler calls any methods that set the last P/Invoke error when marshaling from native to managed or when cleaning up, the value returned by <xref:System.Runtime.InteropServices.Marshal.GetLastWin32Error?displayProperty=nameWithType> and <xref:System.Runtime.InteropServices.Marshal.GetLastPInvokeError?displayProperty=nameWithType> will represent the call in the marshaling or cleanup calls. This can cause errors to be missed when using custom marshalers with P/Invokes with `SetLastError=true`. To preserve the last P/Invoke error, use the <xref:System.Runtime.InteropServices.Marshal.GetLastPInvokeError?displayProperty=nameWithType> and <xref:System.Runtime.InteropServices.Marshal.SetLastPInvokeError?displayProperty=nameWithType> methods in the <xref:System.Runtime.InteropServices.ICustomMarshaler> implementation.
```

Comment:
```suggestion
>  If a custom marshaler calls any methods that set the last P/Invoke error when marshaling from native to managed or when cleaning up, the value returned by <xref:System.Runtime.InteropServices.Marshal.GetLastWin32Error?displayProperty=nameWithType> and <xref:System.Runtime.InteropServices.Marshal.GetLastPInvokeError?displayProperty=nameWithType> will represent the call in the marshaling or cleanup calls. This can cause errors to be missed when using custom marshalers with P/Invokes with <xref:System.Runtime.InteropServices.DllImportAttribute.SetLastError?displayProperty=nameWithType> set to true. To preserve the last P/Invoke error, use the <xref:System.Runtime.InteropServices.Marshal.GetLastPInvokeError?displayProperty=nameWithType> and <xref:System.Runtime.InteropServices.Marshal.SetLastPInvokeError?displayProperty=nameWithType> methods in the <xref:System.Runtime.InteropServices.ICustomMarshaler> implementation.
```